app/views/members/_member_page_header.html.haml

```diff
@@ -13,17 +13,30 @@
           %p.member-data.object-data
             - if member.person.rebellions_fraction
               %span.member-rebellions.object-data-rebellion
-                -if member.person.rebellions_fraction == 0
-                  Never rebels
-                -else
-                  -# TODO: Should this be an absolute count rather than percentage?
-                  -# Maybe it's good to show it as a percentage because it highlights rarity?
-                  Rebels
-                  = fraction_to_percentage_display(member.person.rebellions_fraction)
-                  of the time
-                  -# TODO: add helper tooltip for rebellions
-                  -# link_to "explain...", help_faq_path(anchor: "clarify")
+                - if !member.currently_in_parliament?
```

Comment:
Hi @Anishka0107 ,

Thanks for this PR. I would like to suggest a couple of suggestions. I noticed you duplicated the `if` block. That's usually not very good, because if we need to change the logic of this block, now we will need to do in two places. There is a principle for that if you are interested to read more about it. It's called [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Ruby also can help here. Instead to duplicate the `if` block, you can just added two ifs to add the right messages. For example, you can replace the string `Rebels` for the code below:

```ruby
= "Rebels" if member.currently_in_parliament?
= "Rebelled" if !member.currently_in_parliament?
```
This will print `Rebels` when the `member.currently_in_parliament` and `Rebelled` when the member is not. So then you don't need to duplicate the `if` block :smile: